diff -Nur fusiondirectory.orig/include/class_config.inc fusiondirectory/include/class_config.inc
--- fusiondirectory.orig/include/class_config.inc	2015-10-08 19:51:11.359649464 +0200
+++ fusiondirectory/include/class_config.inc	2015-10-08 19:44:00.651101240 +0200
@@ -1076,6 +1076,9 @@
           if (is_numeric($key)) { // This is not the main tab
             $tabclass = strtoupper($value)."TABS";
             @DEBUG (DEBUG_TRACE, __LINE__, __FUNCTION__, __FILE__, $tabclass, "Adding $class to tab list");
+            if (($tabclass == 'GROUPTABS') && class_available('mixedGroup')) {
+              $tabclass = 'OGROUP-USERTABS';
+            }
             if (!isset($this->data['TABS'][$tabclass])) {
               $this->data['TABS'][$tabclass] = array();
             }
diff -Nur fusiondirectory.orig/include/simpleplugin/class_attribute.inc fusiondirectory/include/simpleplugin/class_attribute.inc
--- fusiondirectory.orig/include/simpleplugin/class_attribute.inc	2015-10-08 19:51:04.631766190 +0200
+++ fusiondirectory/include/simpleplugin/class_attribute.inc	2015-10-08 19:42:45.652394038 +0200
@@ -451,8 +451,12 @@
           if (!is_numeric($key)) {
             $ot = $key;
           }
-          $oinfos = objects::infos($ot);
-          return $oinfos['filter'];
+          try {
+            $oinfos = objects::infos($ot);
+            return $oinfos['filter'];
+          } catch (NonExistingObjectTypeException $e) {
+            return '';
+          }
         },
         array_keys($infos['plObjectType']),
         array_values($infos['plObjectType'])
diff -Nur fusiondirectory.orig/plugins/admin/groups/class_groupManagement.inc fusiondirectory/plugins/admin/groups/class_groupManagement.inc
--- fusiondirectory.orig/plugins/admin/groups/class_groupManagement.inc	2015-10-08 19:50:51.299997482 +0200
+++ fusiondirectory/plugins/admin/groups/class_groupManagement.inc	2015-10-08 20:00:46.393666217 +0200
@@ -75,7 +75,7 @@
                        "alt='"._('Posix')."' title='"._('Edit posix properties')."' ".
                        "name='listing_edit_tab_group_$row' style='padding:1px'/>";
     if ($attrs = $ldap->fetch()) {
-      if (!objects::isOfType($attrs, 'group')) {
+      if (!objects::isOfType($attrs, 'role')) {
         return '<input class="center" type="image" src="geticon.php?context=types&amp;icon=role&amp;size=16" '.
                  'alt="'._('Role').'" title="'._('Edit role properties').'" '.
                  'name="listing_edit_'.$row.'" style="padding:1px"/>';
diff -Nur fusiondirectory.orig/plugins/admin/groups/class_ogroup.inc fusiondirectory/plugins/admin/groups/class_ogroup.inc
--- fusiondirectory.orig/plugins/admin/groups/class_ogroup.inc	2015-10-08 19:51:15.711573960 +0200
+++ fusiondirectory/plugins/admin/groups/class_ogroup.inc	2015-10-08 19:31:59.491541531 +0200
@@ -154,7 +154,7 @@
         'name'        => _('Group'),
         'description' => _('Group'),
         'ou'          => get_ou('ogroupRDN'),
-        'filter'      => 'objectClass=gosaGroupOfNames',
+        'filter'      => 'objectClass=groupOfNames',
         'icon'        => 'geticon.php?context=types&icon=resource-group&size=16',
         'tabClass'    => 'ogrouptabs',
       )),
@@ -253,7 +253,7 @@
     foreach ($res as $data) {
       $ws_dns[] = $data['dn'];
     }
-    $res = get_list("(&(member=*)(objectClass=gosaGroupOfNames)(|(gosaGroupObjects=[W])(gosaGroupObjects=[T])))", "none",
+    $res = get_list("(&(member=*)(|(objectClass=groupOfNames)(objectClass=groupOfNames))(|(gosaGroupObjects=[W])(gosaGroupObjects=[T])))", "none",
                     $base, array("dn","member"), GL_NO_ACL_CHECK | GL_SUBSEARCH);
     $this->used_workstations = array();
     foreach ($res as $og) {
